train.py

```diff
@@ -141,6 +141,10 @@ def train(hyp, opt, device, tb_writer=None):
                         (weights, ckpt['epoch'], epochs))
             epochs += ckpt['epoch']  # finetune additional epochs
 
+        # Logging
+        if wandb and wandb.run is None:
```

Comment:
I left this if statement alone, but was wondering, will `wandb.run` will always be None?

Reply:
Yes. the logic here is that we call `wandb.init()` only if it hasn't been initialized before in the current process. This ensures that user can call `wandb.init()` manually in their module and call train.py as a sub-process.